portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase

```diff
@@ -0,0 +1,96 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the DM Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "DM Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Page");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			DMDocument.tearDownCP();
+
+			DMFolder.tearDownCP();
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+	@description = "Check that is possible to delete a Document and Media"
+	@priority = 4
+	test CanDeleteDM {
+		var dmEntryTitle = "DMTest";
+		var dmFileList = ListUtil.newListFromString("Document_1.txt,Document_2.doc", ",");
+
+		task ("And create 2 Documents and Media (Upload the file)") {
+			var i = 0;
+
+			for (var file : list ${dmFileList}) {
+				var i = ${i} + 1;
+
+				JSONDocument.addFileWithUploadedFile(
+					dmDocumentTitle = "${dmEntryTitle} ${i}",
+					groupName = "Guest",
+					mimeType = "text/plain",
+					sourceFileName = ${file});
+			}
```

Comment:
Hey, thanks for the update, I believe what you could also do is:
```suggestion
		var dmEntryTitle = "DMTest";
		var dmFile = "Document";

		task ("And create 2 Documents and Media (Upload the file)") {
			for (var i : list "1,2") {
				JSONDocument.addFileWithUploadedFile(
					dmDocumentTitle = "${dmEntryTitle} ${i}",
					groupName = "Guest",
					mimeType = "text/plain",
					sourceFileName = "${dmFile}_${i}.txt");
			}
```
But what you did is also correct, I'm just showing the other way to do it :slightly_smiling_face: